root/communication-rmi/src/main/java/at/fhv/itb5c/rmi/client/RMIClient.java

```diff
@@ -27,7 +27,11 @@ public void startUp() {
 	}
 
 	public void close() {
-		throw new UnsupportedOperationException();
+		// Note: This will fail, if something on the outside still holds a reference to _applicationFacadeRMIStub.
```

Comment:
could be solved by not allowing a direct reference to the _applicationFacadeRMIStub